README.md

```diff
@@ -23,27 +23,53 @@ Goldfinger.Builder(context).build()
 #### Check prerequisites
 
 ```java
-if (goldfinger.hasEnrolledFingerprint()) {
+if (goldfinger.canAuthenticate()) {
   /* Authenticate */
 }
 ```
 
-#### Authenticate
+#### Build params
 
 ```java
-goldfinger.authenticate(new Goldfinger.Callback() {
-  @Override
-  public void onError(@NonNull Exception e) {
-    /* Critical error happened */
-  }
+Goldfinger.Params params = new Goldfinger.Params.Builder(activity)
+  .title("Title")
+  .negativeButtonText("Cancel")
+  .description("Description")
+  .subtitle("Subtitle")
+  .build();
```

Comment:
Maybe you could write in readme which params are mandatory and which are optional.